client/state/data-layer/wpcom-http/README.md

```diff
@@ -329,3 +348,35 @@ export default {
 	) ]
 }
 ```
+
```

Comment:
This documentation change deserves its own PR because it is unrelated to `dispatchRequest` accepting an object